crates/bin/prove_block/src/state_utils.rs

```diff
@@ -29,8 +25,7 @@ pub struct FormattedStateUpdate {
 pub(crate) async fn get_processed_state_update(
```

Comment:
You renamed the struct, you can also rename the function to `format_state_update`.